src/dealWizard/stages/proposalStage/proposalStage.ts

```diff
@@ -31,22 +31,14 @@ export class ProposalStage implements IBaseWizardStage {
       this.errors.overview = "Please enter a descriptive overview for your proposal";
     }
 
-    this.wizard.stages.find(stage => stage.name === "Proposal").valid = !Object.keys(this.errors).length;
+    this.wizardService.getCurrentStage(this.wizardManager).valid = !Object.keys(this.errors).length;
 
     return !Object.keys(this.errors).length;
   }
 
   proceed(): void {
```

Comment:
We can move this to the service.  In the HTML, just call `proceed (validateInputs.bind())` and let the service do the work.